webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -0,0 +1,200 @@
+import React, { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Skeleton, Label } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { TableText, TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import {
+  LongArrowAltUpIcon,
+  CheckIcon,
+} from '@patternfly/react-icons';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import { MODULE_STREAMS_KEY } from './ModuleStreamsConstants';
+import { selectModuleStreamStatus } from './ModuleStreamsSelectors';
+import { useBulkSelect } from '../../../../Table/TableHooks';
+import { getHostModuleStreams } from './ModuleStreamsActions';
+import InactiveText from '../../../../../scenes/ContentViews/components/InactiveText';
+import TableWrapper from '../../../../../components/Table/TableWrapper';
+
+export const ModuleStreamsTab = () => {
+  const hostDetails = useSelector(state => selectAPIResponse(state, 'HOST_DETAILS'));
```

Comment:
Extra credit: If you can, now is a good time to use the `useUrlParams` hook: https://github.com/Katello/katello/blob/4a363fd945d56ad48f6a30446d9c16320819d4a5/webpack/components/Table/TableHooks.js#L251

This will allow you to accept URL params and transform them into a pre-filled search box.

See the ErrataTab.js for an example of how to use it. (basically you `useUrlParams` to get the searchParam and then pass that as `initialSearchQuery` to `useBulkSelect`)